### ISSUE TITLE
Widget error

### ISSUE THREAD

**Issue by dimitrios-git** [slytomcat/yandex-disk-indicator#218]
Describe what happened with indicator
The app launches, but it throws errors and it does not appear in the tray
Describe steps to recreate the issue
Just launch the app
Make debugging log:
  1. close indicator if it is running
  2. start indicator in terminal by command line: yandex-disk-indicator -l10
  3. recreate the issue
  4. copy debug output to issue

[user@arch ~]$ yandex-disk-indicator -l10
2020-04-19 18:55:13,142 INFO     yandex-disk-indicator v.1.11.0
2020-04-19 18:55:13,143 DEBUG    Logging level: 10
2020-04-19 18:55:13,145 DEBUG    Config value read as: autostart = True
2020-04-19 18:55:13,145 DEBUG    Config value read as: notifications = True
2020-04-19 18:55:13,145 DEBUG    Config value read as: theme = False
2020-04-19 18:55:13,145 DEBUG    Config value read as: fmextensions = True
2020-04-19 18:55:13,145 DEBUG    Config value read as: daemons = /home/user/.config/yandex-disk/config.cfg
2020-04-19 18:55:13,146 INFO     Config read: /home/user/.config/yd-tools/yandex-disk-indicator.conf
2020-04-19 18:55:13,225 DEBUG    Config value read as: auth = /home/user/.config/yandex-disk/passwd
2020-04-19 18:55:13,225 DEBUG    Config value read as: dir = /home/user/Public/Yandex.Disk
2020-04-19 18:55:13,226 DEBUG    Config value read as: proxy = False
2020-04-19 18:55:13,226 INFO     Config read: /home/user/.config/yandex-disk/config.cfg
2020-04-19 18:55:13,257 INFO     Daemon is already started

(yandex-disk-indicator:16239): Gtk-CRITICAL **: 18:55:13.339: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
2020-04-19 18:55:13,552 DEBUG    Event raised by Timer
2020-04-19 18:55:13,552 INFO     Change event: stat,size,last
2020-04-19 18:55:13,560 DEBUG    Sub-menu 'Last synchronized' has 10 items
2020-04-19 18:55:13,560 INFO     Status: unknown -> idle

(yandex-disk-indicator:16239): Gtk-CRITICAL **: 18:55:13.561: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed

Describe your OS (distribution, version, desktop environment)
Archlinux 5.6.4
GNOME 3.36

**Comment by slytomcat** [comment]
Is the application indicator plugin added to the try?
yandex-disk-indicator uses the app indicator pluging to show its icon.

**Comment by slytomcat** [comment]
I also can see 'Gtk-CRITICAL **: 18:55:13.561: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed' error in the log, but idicator works normally.

**Comment by dimitrios-git** [comment]
Hello, I am sorry but I don't understand your comments. I can confirm that synchronizing works normally, but the indicator is not showing.

**Comment by slytomcat** [comment]
Indicator shows its icon in the App Indicator plugin. The App Indicator plugin have to be added to tray to work. Is your tray have the App Indicator plugin added?

**Comment by dimitrios-git** [comment]
Thanks for the quick reply.
Let me clarify. Since I am on Archlinux, I am using the package from AUR to install this. 
There are two packages there:

https://aur.archlinux.org/packages/yandex-disk-indicator/
this one produces the error that GTK_IS_WIDGET
the AUR package takes care of the dependencies and therefore it installs libappindicator-gtk3.

I also tried that other package (https://aur.archlinux.org/packages/yandex-disk-indicator-git/), which is based on your git, but that won't even start giving me this error:
`python3: can't open file 'indicator.py'": [Errno 2] No such file or directory`

**Comment by slytomcat** [comment]
AUR packages/repos is out of my responsibilities. Sorry.

If all the dependencies are satisfied, then you can install the Indicator from this repo as source and the only thing to do - to run build/install.sh script as root. The indicator is written in Python3 and bash and does't require a building. In this case I can assist you to fix the issue.

But I'm asking not about the way of installation. I'm asking about your desktop configuration.

Desktop tray have to have the special plugin to show the indicator icon. And first of all we have to clarify: is that tray plugin (usually it's called "indicators") exists in your desktop tray bar?

**Comment by dimitrios-git** [comment]
I understand that you cannot assist with the AUR package. However, as you said, the problem is not with the installation. I removed the AUR package and installed it using your script. I get exactly the same error.

About the indicator plugin: Now I understand what you meant. 

I am using GNOME 3.36 and I have installed this extension for the tray icons: https://extensions.gnome.org/extension/2890/tray-icons-reloaded/
It works well with all my other apps. I've also tried different extensions. Nothing works for your app.
